wallet/udb/addressmanager.go

```diff
@@ -425,13 +425,26 @@ func (m *Manager) GetMasterPubkey(ns walletdb.ReadBucket, account uint32) (strin
 
 	// The account is either invalid or just wasn't cached, so attempt to
 	// load the information from the database.
-	row, err := fetchAccountInfo(ns, account, DBVersion)
+	row, err := fetchAccountInfo(ns, account)
 	if err != nil {
 		return "", err
 	}
 
+	var acctRow *dbBIP0044AccountRow
+	switch row.acctType {
+	case actBIP0044:
+		accountID := uint32ToBytes(account)
+		acctRow, err = deserializeBIP0044AccountRow(accountID, row, DBVersion)
```

Comment:
It looks like in all cases, the calls to deserializeBIP0044AccountRow all take the account as a byte slice, which now requires an extra allocation just before.  This would be better if it took the account as a uint32 instead.